mirage/seed_image/catalog_seed_image.py

```diff
@@ -1928,25 +1920,7 @@ def cropPSF(self, psf):
         return psf[nyshift - ydist:nyshift + ydist + 1, nxshift - xdist:nxshift + xdist + 1]
 
     def readPointSourceFile(self, filename):
-        """Read in the point source catalog file
-
-        Parameters:
-        -----------
-        filename : str
-            Filename of catalog file to be read in
-
-        Returns:
-        --------
-        gtab : Table
-            astropy Table containing catalog
-
-        pflag : bool
-            Flag indicating units of source locations. True for detector
-            pixels, False for RA, Dec
-
-        msys : str
-            Magnitude system of the source brightnesses (e.g. 'abmag')
-        """
+        # Read in the point source list
```

Comment:
This looks accidental; PR should be updated to not delete the docstring of this function.